requirements.txt

```diff
@@ -1,3 +1,3 @@
 Scrapy>=1.0.2
 service_identity==1.0.0
-https://github.com/scrapinghub/scrapinghub-entrypoint-scrapy/tarball/7e9190e#egg=sh_scrapy
+scrapinghub-entrypoint-scrapy
```

Comment:
hum, this will be cached during deploy, maybe we should pin the package version?